packages/models-library/src/models_library/projects_nodes_io.py

```diff
@@ -47,6 +49,9 @@ class Config:
 
 
 class BaseFileLink(BaseModel):
+    # TODO: constructor will always cast to str here. We should perhaps set is as str. Actually
+    # if we want to do hash in inputs/outputs ... we should have a single type for identifiers
+    # Recall lru_cache options regarding types!!
     store: Union[str, int] = Field(
```

Comment:
you are right. actually it should be Union[int,str] or else it is useless.

Reply:
I do not want to touch this now here ... until we review the entire project webserver API. I propose to do this in a different PR